packages/react/src/components/HotspotEditorModal/HotspotEditorModal.jsx

```diff
@@ -625,13 +625,7 @@ const HotspotEditorModal = ({
       <div>
         <ContentSwitcher
           onChange={(e) => {
-            switchCurrentType(
-              e.name === 'labels'
-                ? hotspotTypes.TEXT
-                : dynamicHotspotSourceX && dynamicHotspotSourceY
-                ? hotspotTypes.DYNAMIC
-                : hotspotTypes.FIXED
-            );
```

Comment:
Did this ever serve a purpose? I can't figure out exactly what's happening there.

Reply:
Well, setting the `currentType` to `TEXT` or `FIXED` also determines what kind of hotspot that is created when a user ctrl-clicks the image. It is however not possible to create a `DYNAMIC` hotspot so setting that value was not needed. I think I originally added it for consistency, since there are 3 types, but that classification is not perfect either...